logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultChronoLocalDateTimeAssertions.kt

```diff
@@ -14,29 +16,72 @@ class DefaultChronoLocalDateTimeAssertions : ChronoLocalDateTimeAssertions {
         expected: ChronoLocalDateTime<*>
     ): Assertion = container.createDescriptiveAssertion(IS_BEFORE, expected) { it.isBefore(expected) }
 
+    override fun <T : ChronoLocalDateTime<out ChronoLocalDate>> isBefore(
+        container: AssertionContainer<T>,
+        expected: String
+    ): Assertion = container.createDescriptiveAssertion(IS_BEFORE, expected) {
+        it.isBefore(stringToLocalDateTime(expected))
+    }
```

Comment:
please delegate to the overload which expects `ChronoLocalDateTime<*>`
```suggestion
    ): Assertion = container.isBefore(stringToLocalDateTime(expected))
```